controllers/registries/testing/instance.go

```diff
@@ -19,7 +19,7 @@ func CreateInstance(name, namespace string) *v1alpha2.Instance {
 			Type:        "manual",
 			InstanceURL: "https://core.harbor.domain",
 			GarbageCollection: &v1alpha2.GarbageCollection{
-				Cron:         "0 * * * *",
+				Cron:         "0 * * * * *",
```

Comment:
What's up with the new cronjob pattern? Is the sixth segment required, now? How do we handle compatibility in this case?

Reply:
With https://github.com/goharbor/harbor/blob/320c64e4335685952a76c22bcd76e88c611918d8/src/common/utils/utils.go#L299 a custom cron parser was introduced that now requires us to also use the `Second` parameter (Note that the parser is using `Second` and not `SecondOptional`, as suggested in the [guide](https://github.com/robfig/cron#upgrading-to-v3-june-2019) on how to retain the old behaviour). In the latest diff, the cron schedule used to configure Harbor garbage collection will prepend `*` if left unspecified in helm values.